src/test/java/com/example/raisetechcoursetask10/mapper/SkiresortMapperTest.java

```diff
@@ -70,6 +70,7 @@ class FindByIdTest {
     class CreateTest {
         @Test
         @DataSet(value = "datasets/ut/skiresort.yml")
+        @ExpectedDataSet(value = "datasets/ut/create-skiresort.yml", ignoreCols = "id")
         @Transactional
         void 新規スキーリゾートが登録できること() {
             Skiresort skiresort = new Skiresort(4, "湯の丸", "長野県", "急斜面は道路を渡って行かなければならない。");
```

Comment:
登録時点で ID は指定しないようにするのがいいかなと思いました。
```suggestion
            Skiresort skiresort = new Skiresort(0, "湯の丸", "長野県", "急斜面は道路を渡って行かなければならない。");
```
IDはINSERT文を発行して発番されますからね。

Reply:
0だと指定なしなんですね。
ありがとうございます。

Reply:
本来はID指定しないコンストラクタ作っておくのもいいとは思いますね。
```java
    public Skiresort(String name, String area, String customerEvaluation) {
        this.id = 0;
        this.name = name;
        this.area = area;
        this.customerEvaluation = customerEvaluation;
    }
```

Reply:
Skiresortに追記したらいろんなところでエラーが出てしまって・・。
コンストラクタを追記する場所を変更したら成功しました！